.example.env

```diff
@@ -18,3 +18,5 @@ LOGOUT_REDIRECT_URI=http://localhost:3000
 NODE_ENV=local
 PHASE_BANNER_HTML='no messages will be sent'
 PHASE_BANNER_TAG=development
+PHONE_NUMBERS_CONTAINER=phone-numbers
+PHONE_NUMBERS_STORAGE_CONNECTION_STRING="AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;DefaultEndpointsProtocol=http;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;QueueEndpoint=http://127.0.0.1:10001/devstoreaccount1;TableEndpoint=http://127.0.0.1:10002/devstoreaccount1;"
```

Comment:
I think this is the connection string for testing provided my MS, so isn't something to worry about having in the repo?

Reply:
Yeah, it is the Azurite [connection string](https://github.com/azure/azurite#connection-strings).